src/Altinn.App.Core/Features/Signing/SigningService.cs

```diff
@@ -344,11 +344,13 @@ private static async Task<Signee> GetSignee(UserActionContext context)
             case Authenticated.User user:
             {
                 var userProfile = await user.LookupProfile();
+                var orgProfile = await user.LookupSelectedParty();
+
                 return new Signee
                 {
                     UserId = userProfile.UserId.ToString(CultureInfo.InvariantCulture),
                     PersonNumber = userProfile.Party.SSN,
-                    OrganisationNumber = userProfile.Party.OrgNumber,
+                    OrganisationNumber = orgProfile.OrgNumber,
```

Comment:
Is there an org nr here when the logged in user has selected an org during party selection? And if the user has chosen to just represent himself this will be null/empty and that is OK?